meta-lhg/recipes-security/ocdm/ocdmi_git.bb

```diff
@@ -22,7 +28,9 @@ EXTRA_OECONF_append = "${@bb.utils.contains('MACHINE_FEATURES', 'optee', '--enab
 DEPENDS_append = " \
     ${@bb.utils.contains('PACKAGECONFIG','--enable-playready','','openssl',d)} \
     rpcbind \
+    ${@bb.utils.contains('PACKAGECONFIG','--enable-tirpc','libtirpc','',d)} \
```

Comment:
I think instead of '--enable-tirpc', 'rpc' should be mentioned. Also once libtirpc at Line#36 is added in PACKAGECONFIG, not required to add into DEPENDS.

Reply:
Thanks for reviewing, will remove from the DEPENDS line. tirpc does have rpc in the last 3 letters ;-) We can update this to something else, but the ocdmi package config changes have already been merged, so this just aligns with what is already in the configure script etc

Reply:
yeah okay. This line is no effect as libtirpc depends added in Line#36 and openssl always added into DEPENDS as per the line#29, so nothing to worry